### DISCUSSION TITLE
DAI 2023-2024 - Practical Work 2 - Herman Loïc, Stefani Massimo

### DISCUSSION THREAD

**Discussion by massteff** [heig-vd-dai-course/heig-vd-dai-course#157] [Show and tell]
We are going to create the CCP! This has nothing to do with the Chinese Communist Party, but it's the Chess Communication Protocol, capable of playing a game of chess with the OOP chess laboratory. [Here!](https://github.com/heig-lherman/dai-pw02)

**Comment by Lutonite** [comment]
Final commit is [5c9754f](https://github.com/heig-lherman/dai-pw02/commit/5c9754f1e3bde80f9060f1ac3c7ec8339c98e142)
